yarn.lock

```diff
@@ -7746,7 +7746,7 @@ replace-ext@1.0.0:
   resolved "https://registry.yarnpkg.com/replace-ext/-/replace-ext-1.0.0.tgz#de63128373fcbf7c3ccfa4de5a480c45a67958eb"
   integrity sha1-3mMSg3P8v3w8z6TeWkgMRaZ5WOs=
 
-request@^2.79.0, request@^2.88.0, request@^2.88.2:
+request@^2.79.0, request@^2.88.0:
```

Comment:
Did you manually change the `package.json` dependencies? If you did `yarn remove request` I would think this wouldn't be here

Reply:
I did yarn remove

Reply:
wonder why thats left behind then

Reply:
I have no clue, I did yarn remove again and seeing that its not in the package.json it just exited